libsodium-sys/libsodium

```diff
@@ -0,0 +1 @@
+Subproject commit f01d7c128214e8f976889c8093a42d9cae1abe4b
```

Comment:
would it make more sense to "lock" the commit sodiumoxide chooses to link against to tagged releases? https://github.com/jedisct1/libsodium/releases (https://github.com/jedisct1/libsodium/commit/fce6852d64339efa33c0ee4130b3107b888d6067 is the latest, 1.0.10)